TELEMETRY.md

```diff
@@ -6216,7 +6216,7 @@ No properties for event
 
 
 
- Telemetry event sent when an ipywidget version that is not supported is used & we have trapped this and warned the user about it.
+ Telemetry event sent when an ipywidget version that is not supported is used & we have trapped this and warned the user abou it.
```

Comment:
👁️ ✏️

Reply:
I didn't write this comment. It's in index.ts. Which not sure why this wasn't picked up this way before.